wpeverest/everest-forms
# Enhancement - Date drop-down feature added

### All Submissions:






### Changes proposed in this Pull Request:



Added drop down date feature to Date Field.

### How to test the changes in this Pull Request:

1. Create Form with Date Field
2. Select drop-down as date style.
3. Choose various options under Advance Setting.
4. Verify the changed with some entries.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Date drop-down feature added.
